daemon/cmd/daemon_main.go

```diff
@@ -1517,14 +1517,6 @@ func initEnv(cmd *cobra.Command) {
 		}
 	}
 
-	// This is necessary because the code inside pkg/k8s.NewService() for
```

Comment:
This goes away, the BGP LB Announcement features are not coupled to the NodePort configuration.